Release 0.27.0 (#678)
sematic-ai/sematic: docs/upgrades.md

## FOSS to "Enterprise Edition"

> 0. Reach out to Sematic via `support@sematic.dev` to obtain a license for "Sematic EE".

Reviewer: ```suggestion
0. Reach out to Sematic via [support@sematic.dev](mailto:support@sematic.dev) to obtain an Enterprise License.
```

No need for the monospace font for an email.

Author: Done.
